eq-author-api/middleware/auth.test.js

```diff
@@ -0,0 +1,117 @@
+const createAuthMiddleware = require("./auth");
+const jwt = require("jsonwebtoken");
+const uuid = require("uuid");
+
+describe("auth middleware", () => {
+  let logger;
+  let context;
+
+  beforeEach(() => {
+    logger = {
+      error: jest.fn()
+    };
+
+    context = {};
+  });
+
+  it("should export a function", () => {
+    expect(createAuthMiddleware).toEqual(expect.any(Function));
+  });
+
+  it("should create middleware function", () => {
+    const middleware = createAuthMiddleware(logger, context);
+    expect(middleware).toEqual(expect.any(Function));
+  });
+
+  describe("calling the middleware", () => {
+    let req;
+    let res;
+    let next;
+
+    let middleware;
+    beforeEach(() => {
+      req = {
+        header: jest.fn()
+      };
+      res = {
+        send: jest.fn()
+      };
+      next = jest.fn();
+
+      middleware = createAuthMiddleware(logger, context);
+    });
+
+    it("should check the authorization header", () => {
+      req.header.mockImplementation(() => null);
```

Comment:
You can use `mockReturnValue` instead but I think you'll get this by default with your empty `jest.fn()`